fs/romfs/fs_romfs.c

```diff
@@ -578,9 +582,7 @@ static off_t romfs_seek(FAR struct file *filep, off_t offset, int whence)
 
 static int romfs_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
 {
-  FAR struct romfs_mountpt_s *rm;
   FAR struct romfs_file_s    *rf;
```

Comment:
remove th extra space before rf

Reply:
the style in this file is to align all local variables, in every function. better to keep the original style

Reply:
done